index.js

```diff
@@ -1,41 +1,40 @@
+'use strict';
+const Options = require('./options')
 
-var props = [
-  'name',
-  'message',
-  'stack',
-  'type',
-]
-
-module.exports = function () {
+module.exports = function(opts) {
+  var options = Options.from(opts);
   return function* jsonErrorHandler(next) {
     var status
     try {
-      yield* next
+      yield * next
 
       status = this.response.status
       // future proof status
-      if (!status || (status === 404 && this.response.body == null)) this.throw(404)
+      if (!status || (status === 404 && this.response.body == null)) {
+        this.throw(404)
+      }
     } catch (err) {
       // set body
-      var obj =
-      this.response.body = {}
+      var obj = this.response.body = {}
 
       // set status
       status =
-      this.response.status =
-      err.status = err.status || 500
+        this.response.status =
```

Comment:
Agreed. I'll revert those. But since you are _not enforcing_ any linting (because you don't have any tool configured for that) nor have any clear reference for contribution guidelines on this repository, you cannot expect contributors to follow rules you don't explicitly hang on to.